packages/shared/src/components/Feed.tsx

```diff
@@ -176,6 +186,7 @@ export default function Feed<T>({
       },
     },
   );
+  const canFetchMore = allowFetchMore ?? queryCanFetchMore;
```

Comment:
Might be missing something, but from usage, if query `canFetchMore` is false why would we overwrite it since there should be no more pages to fetch?

Reply:
Technically there are (eg api will say yes more pages)

Reply:
Does `hasNextPage` not mean anything then? Maybe we can instead expose `getNextPageParam` and `getPreviousPageParam`, it will make it more versatile in the future.

Reply:
They different things basically since we didn't want to actually load 2 posts.
So the FE shows only 2 posts with it's own pagination, but internally loads all 10 pages at once (As a optimization)

Keep in mind ideally this whole thing would have been it's own component, but we decided to re-use feed since it has all the card logic

Hope that makes sense?

Reply:
> So the FE shows only 2 posts with it's own pagination, but internally loads all 10 pages at once (As a optimization)

You mean on feed/api side cache, frontend loads page per page as far as I know?

Reply:
I'll show you in our 1:1 quickly it's a bit difficult to explain :)